src/engine/RuntimeInformation.cpp

```diff
@@ -122,8 +124,11 @@ double RuntimeInformation::getOperationTime() const {
     // `totalTime_`. That's why we skip such children in the following loop.
     auto timesOfChildren =
         children_ | std::views::transform(&RuntimeInformation::totalTime_);
-    return totalTime_ -
-           std::accumulate(timesOfChildren.begin(), timesOfChildren.end(), 0.0);
+    // Prevent "negative" computation times in case totalTime_ was not
+    // computed for this yet.
+    constexpr auto zero = std::chrono::milliseconds::zero();
+    return std::max(zero, -std::reduce(timesOfChildren.begin(),
```

Comment:
Use `0ms` literal.
And the `std::reduce` with the double negative is really unreadable here, please use the direct calculation as before....